README.md

```diff
@@ -34,8 +34,11 @@ You will also need a copy of the [App Engine SDK](https://developers.google.com/
 
 If you want to make changes to the source, you are going to need to run an automated test suite, and for that you will also need [phantomjs](http://phantomjs.org/). Have a look at the testing section for more information.
 
-Note: Java 8 will work locally, and it is also supported by AppEngine, but you may need to [specify your runtime](https://cloud.google.com/appengine/docs/standard/java/runtime-java8) if you want to use appspot.
-Also note that certain Java 8 features, such as lambda expressions, are not supported on Android, so please don't use them in your changes to the source code.
+Note 1: If you are working on a 64-bit linux system, you need to install 32-bit version of: glibc(to get a 32-bit version of ld-linux.so), zlib and libstdc++.
+If you are on a Debian-based distribution(Ubuntu), use: sudo apt-get install libc6:i386 zlib1g:i386 libstdc++6:i386.
```

Comment:
Two minor formatting things:

1. Have two line breaks between these lines. When the markdown renders, it currently ends up in a single paragraph. I think it would look better if the different conditions were on separate lines.

2. Use single backquotes (\`) around the sudo command so that it formats as code.